setup.py

```diff
@@ -1,7 +1,38 @@
-#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 # Licensed under a 3-clause BSD style license, see LICENSE.
 
+from __future__ import print_function
+
+import os
+
+import setuptools.command.sdist
 from setuptools import setup
 
-setup()
+datafile = "src/skhep_testdata/data"
+data_ex = {".root", ".lhe"}
+
+data_files = {n for n in os.listdir(datafile) if any(n.endswith(ex) for ex in data_ex)}
+
+if data_files:
+    with open(os.path.join(datafile, "file_list.txt"), "w") as f:
```

Comment:
This could probably be more elegant, but should suffice for now for a test data repo.

Reply:
Yes.